src/pages/_app.tsx

```diff
@@ -94,7 +95,7 @@ function UIProvider({ children }: React.PropsWithChildren<{}>) {
 
 const config = {
   autoConnect: true,
-  connectors: [new MetaMaskConnector(), new JoyIdConnector()],
+  connectors: [new MetaMaskConnector, new JoyIdConnector()],
```

Comment:
Looks like the parentheses are missing

Reply:
remove this due to the hotel network not stable, will fix